packages/database/schema.prisma

```diff
@@ -53,52 +53,52 @@ model Webhook {
   @@index([environmentId])
 }
 
-model Attribute {
-  id               String         @id @default(cuid())
-  createdAt        DateTime       @default(now()) @map(name: "created_at")
-  updatedAt        DateTime       @updatedAt @map(name: "updated_at")
-  attributeClass   AttributeClass @relation(fields: [attributeClassId], references: [id], onDelete: Cascade)
-  attributeClassId String
-  person           Person         @relation(fields: [personId], references: [id], onDelete: Cascade)
-  personId         String
-  value            String
+model ContactAttribute {
+  id             String              @id @default(cuid())
+  createdAt      DateTime            @default(now()) @map(name: "created_at")
+  updatedAt      DateTime            @updatedAt @map(name: "updated_at")
+  attributeKey   ContactAttributeKey @relation(fields: [attributeKeyId], references: [id], onDelete: Cascade)
+  attributeKeyId String
+  contact        Contact             @relation(fields: [contactId], references: [id], onDelete: Cascade)
+  contactId      String
+  value          String
 
-  @@unique([personId, attributeClassId])
+  @@unique([contactId, attributeKeyId])
 }
 
-enum AttributeType {
-  code
-  noCode
-  automatic
+enum ContactAttributeType {
+  default
+  custom
 }
 
-model AttributeClass {
+model ContactAttributeKey {
```

Comment:
please add a field
```
isUnique       Boolean  @default(false)
```
to indicate if an attribute key needs to be unique or not. This will be enforced by application logic.